public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -156,7 +156,9 @@ export class TimeSrv {
     this.refresh = getRefreshFromUrl({
       params: paramsJSON,
       currentRefresh: this.refresh,
-      refreshIntervals: this.dashboard?.timepicker?.refresh_intervals,
+      refreshIntervals: Array.isArray(this.dashboard?.timepicker?.refresh_intervals)
+        ? this.dashboard?.timepicker?.refresh_intervals
+        : undefined,
```

Comment:
It's a good idea to be explicit about which value you want to default to as soon as you use `?`, right?

Reply:
@tskarhed do you mean like removing the optional chaining from the first option? 🤔 

e.g.
```javascript
      refreshIntervals: Array.isArray(this.dashboard?.timepicker?.refresh_intervals)
        ? this.dashboard.timepicker.refresh_intervals
        : undefined,
```